Livecheckables/mupen64plus.rb

```diff
@@ -0,0 +1,4 @@
+class Mupen64plus
+  livecheck :url   => "https://github.com/mupen64plus/mupen64plus-core/releases",
```

Comment:
```suggestion
  livecheck :url   => "https://github.com/mupen64plus/mupen64plus-core/releases/latest",
```

Normally we want to use the Git tags rather than a release page but it makes sense when we need to respect GitHub latest/pre-release labels. However, the "latest" release may be pushed off the first page of releases, so we want to simply use the URL for the latest release above (which points to the page for the latest release).